shared/core/src/listings.ts

```diff
@@ -3,7 +3,8 @@ import { Address } from "./general"
 import { Preference } from "./preferences"
 
 export enum AttachmentType {
```

Comment:
this should really be ApplicationType, no?

When we have applications in the system, nothing will be attached, per se.

Reply:
but both of the enum options look right to me.

Reply:
Well this is related to the earlier work we did to support file "attachments" and multiple versions at that (like PDFs for English, Spanish, etc.) and in lieu of using a third party service this is the current arrangement. It's not really an application type per se.